packages/react-composites/src/composites/CallComposite/components/buttons/Devices.tsx

```diff
@@ -32,6 +33,13 @@ export const Devices = (props: {
   const permissions = _usePermissions();
   /* @conditional-compile-remove(rooms) */
   const locale = useLocale();
+  /* @conditional-compile-remove(rooms) */
+  const OnlyManageSpeakers = !permissions.microphoneButton && !permissions.cameraButton;
```

Comment:
nit: regular variables that are primitives are camelCase
```suggestion
  const onlyManageSpeakers = !permissions.microphoneButton && !permissions.cameraButton;
Write
```

Reply:
Done